### PR TITLE
Feature/ka 33 auto test code

### PR DESCRIPTION
**Pull request by hsperker**
add build and report workflow showing test reports by using test reports of Java and TypeScript

### CODE REVIEW

File: .github/workflows/build_report.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    - name: Checkout Sources
+      uses: actions/checkout@v3
+    - name: Setup Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: 18
+    - run: npm install -g jest
+    - run: npm run init:dev
+    - run: npm test
+    - name: Test Report TypeScript
```
Comment (drcgjung) [inline .github/workflows/build_report.yml:48]: The plugin uploads the resulting MD to an "arbitrary" run attached to the same commit.
Maybe we should (unless the plugin is extensible or debugged) merge this workflow with the codeql.yml (who will mostly run in parallel to this one).

**Review by drcgjung — APPROVED** [review]
LGTM

File: .github/workflows/build_report.yml (hunk 1 above)
Comment (hsperker) [reply .github/workflows/build_report.yml:48]: @drcgjung how about merging with codeql but placing the build / test at the beginning for give developers the fast feedback that they will need during development?

**Review by drcgjung — APPROVED** [review]
Supercool!

File: .github/workflows/build_report.yml (hunk 1 above)
Comment (drcgjung) [reply .github/workflows/build_report.yml]: very very neat

**Pull request merged**